core/templates/App.ts

```diff
@@ -15,7 +15,8 @@
 /**
  * @fileoverview Initialization and basic configuration for the Oppia module.
  */
-
+require('@angular/material/prebuilt-themes/deeppurple-amber.css');
+import 'hammerjs';
```

Comment:
Why is this added?

Reply:
angular material (v8) requires hammerjs for gesture recognition. You can look at point 2 [here](https://v8.material.angular.io/guide/getting-started#install-angular-material). But angular material (v9) onwards doesn't require hammerjs. So mostly this is will be temporary.

Reply:
Okay, please add a TODO here referencing #9172 and containing explanation why is this needed.

Reply:
Done!